From c6ec24cf147a4cdb3ecc8c40be513a2af61848d2 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:48:52 -0400
Subject: [tools] perf: Export setup_list

Message-id: <1408953210-26343-45-git-send-email-jolsa@redhat.com>
Patchwork-id: 88768
O-Subject: [PATCH RHEL7.1 BZ1133083 044/322] perf tools: Export setup_list
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 3bfe5f81fc8f87bf822f3da36927cfc549f3b3db
Author: David Ahern <dsahern@gmail.com>
Date: Mon Nov 18 13:32:48 2013 -0700

description
===========
Used in upcoming patches (perf sched timehist command).
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/symbol.c b/tools/perf/util/symbol.c
index f55c18d..ce9ce10 100644
--- a/tools/perf/util/symbol.c
+++ b/tools/perf/util/symbol.c
@@ -1749,7 +1749,7 @@ out_fail:
  return -1;
 }
 
-static int setup_list(struct strlist **list, const char *list_str,
+int setup_list(struct strlist **list, const char *list_str,
         const char *list_name)
 {
  if (list_str == NULL)
diff --git a/tools/perf/util/symbol.h b/tools/perf/util/symbol.h
index ad13c5d..f1031a1 100644
--- a/tools/perf/util/symbol.h
+++ b/tools/perf/util/symbol.h
@@ -274,4 +274,7 @@ void kcore_extract__delete(struct kcore_extract *kce);
 int kcore_copy(const char *from_dir, const char *to_dir);
 int compare_proc_modules(const char *from, const char *to);
 
+int setup_list(struct strlist **list, const char *list_str,
+        const char *list_name);
+
 #endif /* __PERF_SYMBOL */
-- 
1.7.1